mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -483,7 +494,6 @@ class ALSModel private[ml] (
 
   /**
    * Blockifies factors to improve the efficiency of cross join
-   * TODO: SPARK-20443 - expose blockSize as a param?
    */
   private def blockify(
       factors: Dataset[(Int, Array[Float])],
```

Comment:
Do we still need default blockSize in this method?

Reply:
You are right. No need to have the default blockSize any more. I will update the code. Thanks!